helm/vitess/templates/_vttablet.tpl

```diff
@@ -306,6 +306,16 @@ spec:
 - name: mysql
   image: {{.mysqlImage | default $defaultVttablet.mysqlImage | quote}}
   imagePullPolicy: Always
+  readinessProbe:
+    exec:
+      command: ["mysqladmin", "ping", "-uroot", "--socket=/vtdataroot/tabletdata/mysql.sock"]
+    initialDelaySeconds: 60
+    timeoutSeconds: 10
+  livenessProbe:
```

Comment:
On second thought, maybe it's more dangerous than helpful to have a livenessProbe on mysqld since it's got a lot more in-memory state than the rest of our processes (you lose a lot on restart). What do you think about having only the readinessProbe here? It seems like a safer default.

Reply:
That's probably a good choice